### ISSUE TITLE
[Visual Studio Code Recent Projects] stopped being able to load vscode projects

### ISSUE THREAD

**Issue by matehat** [raycast/extensions#1624]
# Extension – [Visual Studio Code Recent Projects](https://github.com/raycast/extensions/tree/5fad3eeb634eb9a5a3493fe9c0cebe036a5291a1/extensions/visual-studio-code-recent-projects/)
Author:

## Description

Command can't load vscode recent workspaces since latest VSCode update.
<img width="828" alt="CleanShot 2022-05-07 at 08 18 52@2x" src="https://user-images.githubusercontent.com/1283/167254145-2a69909a-ce1d-4ddd-8252-6d4d520da4fd.png">

## Steps To Reproduce

1. Upgrade VSCode
2. Run the command

## The current behavior

<img width="828" alt="CleanShot 2022-05-07 at 08 18 52@2x" src="https://user-images.githubusercontent.com/1283/167254161-ebf9e57c-49fa-4734-aa63-a3fbfdbf3def.png">

## The expected behavior

Works as intended

### Raycast version

1.34.1

**Comment by cy-98** [comment]
I try to reinstall, and it also not work. 
![image](https://user-images.githubusercontent.com/43307949/167257000-d9146631-1776-4c8d-841e-58ac466a0c29.png)

**Comment by cy-98** [comment]
source code: line 26
```ts
const appKey: string = appKeyMapping[build] ?? appKeyMapping.Code;
```
I found the appKey to find `vs code` is not correct in my macOS by debugging.

I found the application folder, and got the key is 'Code'.

source code: line 28

I add a 'Code'.
```ts
const STORAGE = `${homedir()}/Library/Application Support/${build || 'Code'}/User/globalStorage/alefragnani.project-manager`;
```

Now, It's work for me !

**Comment by cy-98** [comment]
And the API `getPreferenceValues` got `{}`, then the build is "undefined" . Why does it happened ?

**Comment by cy-98** [comment]
I create an extension, and copy the `project manager`'s source code, and run dev.
The origin extension  works now .

**Comment by cy-98** [comment]
It can find recent projects in Code/User/globalStorage/storage.json

**Comment by cy-98** [comment]
#1630

**Comment by cy-98** [comment]
resolved, and new version published.

**Comment by matehat** [comment]
Awesome thanks!

**Comment by pernielsentikaer** [comment]
Resolved in #1630
